exercises/clock/canonical-data.json

```diff
@@ -213,7 +213,12 @@
           "minute": 1,
           "add": 3500,
           "expected": "11:21"
-        },
+        }
+      ]
+    },
+    {
+      "description": "Subtract minutes",
+      "cases": [
         {
           "description": "subtract minutes",
           "property": "add",
```

Comment:
We still had subtract vs add then. Personally I do not have a strong opinion on this either, since subtraction is just adding a negativ number.

But I try to look from a beginners view here, who probably will feel dizzy at first, when we ask to subtract amount x by telling him to add amount -x.